samples/sid_end_device/src/cli/sbdt_shell.c

```diff
@@ -0,0 +1,790 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "sid_pal_assert_ifc.h"
+#include "sid_pal_storage_kv_ifc.h"
+#include "zephyr/sys/crc.h"
```

Comment:
minor: althougt this in not a part of the ncs coding standerd, I would suggest to use `#include ""` only for current dictionary files, in other cases, especialy zephyr modules header, please use `#include <>`

Reply:
OK, This style with "" on include is a result of autofill in the editor, I reviewed all includes in the PR and now they should be fine.